doc/configure_tls_protocol_version_and_ciphers.md

```diff
@@ -0,0 +1,136 @@
+# How to configure the Azure IoT C SDK TLS platforms to disable TLS 1.0 and TLS 1.1
```

Comment:
nit: Make large sections ## and sub-sections ###, then have a title (name of file is fine) with summary

Reply:
Deal. Done.